__tests__/app.test.js

```diff
@@ -0,0 +1,60 @@
+const seed = require("../db/seeds/seed.js");
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+
+// console.log(process.env, "<<< process.env");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api", () => {
+  test("GET: 200 response", () => {
+    return request(app)
+      .get("/api")
+      .then((response) => {
+        console.log(response.body, "GET /api response");
+        expect(response.body).toEqual({ msg: "Hello World!" });
+      });
+  });
+});
+describe("GET /api/topics", () => {
+  test("Responds with an array", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const topics = body.topics;
+        expect(Array.isArray(topics)).toBe(true);
+      });
+  });
+  test("Responds with an array of topics with slug & description props", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        console.log(body, "GET /api response");
+        const topics = body.topics;
+        expect(topics).toEqual([
+          { slug: "mitch", description: "The man, the Mitch, the legend" },
+          { slug: "cats", description: "Not dogs" },
+          { slug: "paper", description: "what books are made of" },
+        ]);
+      });
+  });
+  test("404: Responds with 404 with invalid route/endpoint/path", () => {
+    return request(app)
+      .get("/api/toptrumps")
+      .expect(404)
+      .then((response) => {
+        console.log(response, "<<< response message");
+        expect(response.body.msg).toBe("Route/endpoint not found");
+      });
+  });
```

Comment:
I would remove console logs, otherwise good testing here 👍 

I would say the first test checking it's an array is inferred anyway in the second test where you are checking topics to equal an array, so not completely necessary